packages/react-interactions/events/src/dom/__tests__/Drag-test.internal.js

```diff
@@ -188,18 +186,7 @@ describe('Drag event responder', () => {
     const mouseUpEvent = document.createEvent('MouseEvents');
     mouseUpEvent.initEvent('mouseup', true, true);
     divRef.current.dispatchEvent(mouseUpEvent);
-    expect(events).toHaveLength(20);
-    expect(events).toEqual(
-      expect.arrayContaining([
-        expect.objectContaining({
-          diffX: 2,
-          diffY: 2,
-        }),
-        expect.objectContaining({
-          diffX: 21,
-          diffY: 21,
-        }),
-      ]),
-    );
+    expect(events).toHaveLength(0);
+    expect(events).toEqual([]);
```

Comment:
Nor should this test.

Reply:
These 2 assertions are the same aren't they?

Reply:
These tests are removed now anyway, so it doesn't matter :)